docker/build-image.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+#  Copyright (C) 2018-2019 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+USERNAME=usdotfhwastol
+
+cd "$(dirname "$0")"
+IMAGE=$(./get-package-name.sh | tr '[:upper:]' '[:lower:]')
```

Comment:
Should be `IMAGE="$(./get-image-name.sh)"`, see the CARMAPlatform `docker/build-image.sh`. Which repository/commit did you copy these files from? The latest version of CARMAPlatform seems to have them correct.